migrations/env.py

```diff
@@ -0,0 +1,70 @@
+import pyslackersweb
+
+from logging.config import fileConfig
+from sqlalchemy import pool, create_engine
+from alembic import context
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
```

Comment:
I think this is typically where I see the path explicitly set for alembic and the metadata set.

I’d like to double check that before we merge anything

Reply:
Yep you are right. Since we don't have any model using the db right now I didn't look into the autogeneration of migration.